src/main/java/com/google/api/codegen/py/PythonGapicContext.java

```diff
@@ -276,6 +294,60 @@ private String returnTypeComment(Method method, PythonImportHandler importHandle
         "\n\nRaises:\n  :exc:`google.gax.errors.GaxError` if the RPC is aborted.");
     return pythonCommon.convertToCommentedBlock(contentBuilder.toString());
   }
+  
+  private String methodSnippet(Method method, PythonImportHandler importHandler) {
+    Interface service = (Interface) method.getParent();
+    List<String> importStrings = new ArrayList<>();
+    importStrings.add(
+        PythonImport.create(
+                ImportType.APP,
+                method.getFile().getProto().getPackage()
```

Comment:
BTW, I'm doing the import so that we can eventually get Sphinx doctest working. There are a few other issues with that, but I think they might be out of scope for this PR (I'm now running into what appears to be a surface bug, although it could be user error).